app/controllers/plans_controller.rb

```diff
@@ -309,11 +313,18 @@ def public_export
     @a_s_ids = Question.where(id: @a_q_ids).pluck(:section_id).uniq
     a_p_ids = Section.where(id: @a_s_ids).pluck(:phase_id).uniq
     @phases = Phase.includes(sections: :questions).where(id: a_p_ids).order(:number)
+    # name of owner and any co-owners
     @creator_text = @plan.owner.name(false)
     @plan.roles.administrator.not_creator.each do |co_owner|
       @creator_text += ", " + co_owner.name(false)
     end
-    @affiliation = @plan.owner.org.abbreviation
+    # Org name of plan owner
+    @affiliation = @plan.owner.org.name
+    # set the funder name
+    @funder = @plan.template.org.funder? ? @plan.template.org.name : nil
```

Comment:
should we use `@plan.funder` here instead? The create plan process auto-populates that field now but is likely not applicable to existing data. I'll make a note in the ticket to have @sjDCC and @stephaniesimms review the different scenarios in UAT